frontend/src/scenes/funnels/FunnelViz.tsx

```diff
@@ -156,7 +156,7 @@ export function FunnelViz({
         ) : null
     }
     if (featureFlags[FEATURE_FLAGS.FUNNEL_BAR_VIZ] && filters.display == ChartDisplayType.FunnelsTimeToConvert) {
-        return timeConversionBins && timeConversionBins.length > 0 ? <FunnelHistogram /> : null
+        return timeConversionBins?.bins && timeConversionBins?.bins.length > 0 ? <FunnelHistogram /> : null
```

Comment:
Nit: `timeConversionBins?.bins?.length ? <FunnelHistogram /> : null` should work